### PR TITLE
Removed packet length from proto

### PR DESCRIPTION
**Pull request by kingshukdev**
Removed packet length from the proto.

### CODE REVIEW

File: manifests/overlays/e2e/kustomization.yaml [hunk 1]
```diff
@@ -2,6 +2,6 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: networkop/meshnet
-  newTag: 9b9c83b-dirty
```
Comment (apratimmukherjee) [inline manifests/overlays/e2e/kustomization.yaml:5]: Any reason we are seemingly going to an older version ?

**Review by apratimmukherjee — APPROVED** [review]
Looks good to me. One query related to change of meshnet version to lower version .

File: manifests/overlays/e2e/kustomization.yaml (hunk 1 above)
Comment (marcushines) [reply manifests/overlays/e2e/kustomization.yaml:5]: why is that older? isn't that just the sha of the image?

File: daemon/grpcwire/grpcwire.go [hunk 2]
```diff
@@ -409,17 +409,16 @@ func RecvFrmLocalPodThread(wire *GRPCWire) error {
...
 			This logic needs to be better, take the interface MTU not hardcoded value of 1518.
 			This is a very unusual condition to receive an packet from the pod with size > MTU. This can only happens if
 			things gets really messed up.   */
-			if payload.FrameLen > 1518 {
+			if len(data) > 1518 {
 				pktType := DecodePkt(payload.Frame)
-				log.Infof("Daemon(RecvFrmLocalPodThread): Dropping:unusually large packet received from local pod. size: %d, pkt:%s", payload.FrameLen, pktType)
+				log.Infof("Daemon(RecvFrmLocalPodThread): Dropping:unusually large packet received from local pod. size: %d, pkt:%s", len(data), pktType)
 			}
 			ok, err := (wireClient).SendToOnce(ctx, payload)
```
Comment (marcushines) [inline daemon/grpcwire/grpcwire.go:423]: ok, err := wireClient.SendToOnce(ctx, payload)

Comment (kingshukdev) [reply daemon/grpcwire/grpcwire.go]: https://github.com/open-traffic-generator/meshnet-cni/pull/5/commits/dba57dec4d6d8fd458f649767bbb4ecff41a0309

**Pull request merged**